.travis.yml

```diff
@@ -2,18 +2,39 @@ sudo: false
 language: php
 php:
   - 7.2
-
-branches:
-  only: 
-    - master
+services:
+  - docker
+addons:
+  apt:
+    packages:
+      - docker-ce
+env:
+  - DOCKER_COMPOSE_VERSION=1.23.2
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - "$HOME/.npm"
 
 before_install:
-  - git config --global github.accesstoken $GITHUB_OAUTH_TOKEN
-  - composer config -g github-oauth.github.com $GITHUB_OAUTH_TOKEN --no-interaction
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - nvm install 8
+  - nvm use 8
+
+install:
+  - npm ci
+  - ( cd plugins/versionpress && composer install )
```

Comment:
`composer install` runs as part of init-dev, doesn't it?

Reply:
Changed to run init-dev in the install phase instead of using composer install

Reply:
`init-dev` runs as a post-install lifecycle script, it's not necessary to call it manually. See [Travis logs](https://travis-ci.org/versionpress/versionpress/builds/511671083#L531-L648).